### PR TITLE
Bug 1613800 - part 3 to 5: Add tests for missing mobile products and remove old key

### PR DESCRIPTION
**Pull request by JohanLorenzo**
Follows https://github.com/mozilla-releng/scriptworker/pull/449 up. Depends on https://github.com/mozilla-mobile/android-components/pull/6698.

### CODE REVIEW

**Comment by JohanLorenzo** [comment]
I think this PR is ready to reviewed. The tests are failing because I hammered the Github API with many pushes. I'll rerun them in a few hours. I don't foresee any failures.

**Comment by JohanLorenzo** [comment]
Ah crap, it turns out we need a Github token if we want to run all tests.

**Comment by escapewindow** [comment]
> Ah crap, it turns out we need a Github token if we want to run all tests.

We're on TC now. We may want to put a token in tc secrets.

**Comment by JohanLorenzo** [comment]
Depends on https://phabricator.services.mozilla.com/D77797. I manually added the scope to get these tasks green, for now. 

I'd like to get this reviewed once more to make sure I didn't mess up anything secret-wise. r? @escapewindow

File: tests/test_production.py [hunk 1]
```diff
@@ -53,11 +54,22 @@ def build_config(override, basedir):
     return config
 
 
+async def _get_github_token():
```
Comment (escapewindow) [inline tests/test_production.py:57]: We may end up tweaking this to make testing locally easier, but I think this works for now.

File: .taskcluster.yml [hunk 2]
```diff
@@ -37,8 +37,12 @@ tasks:
...
            # See https://github.com/mozilla-releng/scriptworker/issues/415
            ['py38', 'python:3.8', {NO_TESTS_OVER_WIRE: '1'}, []],
-           ['py37-cot', 'python:3.7', {NO_CREDENTIALS_TESTS: '1'}, []],
-           ['py38-cot', 'python:3.8', {NO_CREDENTIALS_TESTS: '1'}, []],
+           ['py37-cot', 'python:3.7', {
+                NO_CREDENTIALS_TESTS: '1'
+            }, ['secrets:get:repo:github.com/mozilla-releng/scriptworker:github']],
+           ['py38-cot', 'python:3.8', {
+                NO_CREDENTIALS_TESTS: '1',
+            }, ['secrets:get:repo:github.com/mozilla-releng/scriptworker:github']],
```
Comment (escapewindow) [inline .taskcluster.yml:45]: Hm. Now that we have secrets, we could add integration tests.
This is a comment for future enhancements (for anyone to do, not necessarily you), and definitely not a blocker for this PR :)

**Pull request merged**